src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -64,10 +85,10 @@ class CosmasGoogleCloudService(private val bucketName: String,
             return
         }
         try {
-            project.forEach { (fileId, file) ->
+            for ((fileId, fileStorage) in project) {
```

Comment:
You know, you call it `fileStorage` here and `fileInformation` in `createPatch` method. Please choose a single name, otherwise we're all going to suffer from light schizophrenia. 
Usually if class is called TheThing then you call variables `thatThing`, `thisThing`, `myThing` etc. However, as I wrote in one of my essays, I personally don't like the name `FileInformation` because it basically means nothing. Perhaps `FileVersion` with `contents` and `patches` fields is more appropriate.

Reply:
ok, change to FileVersion